src/country_workspace/contrib/kobo/forms.py

```diff
@@ -3,12 +3,6 @@
 
 class ImportKoboForm(forms.Form):
     batch_name = forms.CharField(required=False, help_text="Label for this batch")
-    country_code = forms.CharField(
-        required=False,
```

Comment:
Have you checked how it is managed in HOPE? I'm not sure we are relying on country code right now.

Reply:
not sure about this. Please add a note here about how this will work with Kobo, how we will manage existing forms and what changes are needed in the various systems. HOPE BusinessArea does not represent a country, and it does not have any country attribute. (cc @ntrncic, @domdinicola).